### PR TITLE
Integrating CPUC Agnostic Tools

### PR DESCRIPTION
**Pull request by claalmve**
Integrating back-end for computing agnostic tools for PGE/SCE. All functionality lives in `explore/agnostic.py`.

### CODE REVIEW

**Comment by vicford** [comment]
What is the difference between `gwl_1850-1900ref.csv` (which has 828 rows) and `gwl_1850-1900ref (2).csv` (which has 888) rows? 
- I can see that the order is slightly different
- Before this branch gets merged in `gwl_1850-1900ref (2).csv` must be renamed for clarity, and whether or not it is intended to replace `gwl_1850-1900ref.csv`

**Comment by Tianchi-Liu** [comment]
> What is the difference between `gwl_1850-1900ref.csv` (which has 828 rows) and `gwl_1850-1900ref (2).csv` (which has 888) rows?
> 
> * I can see that the order is slightly different
> * Before this branch gets merged in `gwl_1850-1900ref (2).csv` must be renamed for clarity, and whether or not it is intended to replace `gwl_1850-1900ref.csv`

`gwl_1850-1900ref.csv` is the original lookup table. `gwl_1850-1900ref (2).csv` is a debugged version used by only the agnostic tools right now. Eighty out of its 888 rows contain info about the 80 simulations used by the agnostic tools. It is intended to replace the original table, but ideally only after it’s implications to the other tools are understood. I am working on this. How about renaming `gwl_1850-1900ref (2).csv` to `gwl_1850-1900ref_agnostic_tools.csv` in the mean time?

**Comment by vicford** [comment]
> How about renaming `gwl_1850-1900ref (2).csv` to `gwl_1850-1900ref_agnostic_tools.csv` in the mean time

This is very clear, thank you!

File: climakitae/explore/agnostic.py [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+    """
+    return lambda scenario="ssp370", warming_level=None, year=None: find_wl_or_time(
+        lookup_tables, scenario, warming_level, year
+    )
+
+
+##### TASK 2 #####
+
+# Making pre-determined metrics
+metrics = {
```
Comment (claalmve) [inline climakitae/explore/agnostic.py:279]: Is there a better place to put this?

**Comment by claalmve** [comment]
[GWL Function - LOCA & WRF, Part 1](https://trello.com/c/j9xas1ZP/519-gwl-function-loca-wrf-part-1)

**Comment by Tianchi-Liu** [comment]
[Agnostic tools improvement - Tianchi](https://trello.com/c/cWBUbUbT/534-agnostic-tools-improvement-tianchi)

**Pull request merged**